### ISSUE TITLE
Clarification about code

### ISSUE THREAD

**Issue by fatbao55** [mks0601/Hand4Whole_RELEASE#79]
Hi authors, 

Thanks for the great work! I noticed that the bbox center coordinates were divided by 8, where did this number come from?

https://github.com/mks0601/Hand4Whole_RELEASE/blob/afcbdf448e5ceeaad6da33776b68c496b4a3edde/common/nets/module.py#L134-L136

And why is `/ 8` not applied to the `lhand_size`, `rhand_size`, and `face_size` too?

**Comment by mks0601** [comment]
Hi, that is to restore 8x times upsampled feature map size due to the deconv layer of this line
https://github.com/mks0601/Hand4Whole_RELEASE/blob/afcbdf448e5ceeaad6da33776b68c496b4a3edde/common/nets/module.py#L119

**Comment by mks0601** [comment]
Before the downsampling, `_center` are defined in the upsampled feature map space as they are obtained by the soft-argmax.
`_size` are defined in original feature map space (before upsampling) as GTs are defined in the original space.

**Comment by mks0601** [comment]
`_center` is dependent on the shape of the feature map as they are obtained by the soft-argmax function. Taking an example of argmax function, coordinates from argmax function is dependent on the shape of the feature map.
`_size` is independent on the shape of the feature map as they are directly regressed. Any numbers could be regressed, and as it is supervised with `_size`, defined in the original (before the upsampling) space, `_size` does not need to be divided by 8.

**Comment by mks0601** [comment]
Hi, I think there is a still some confusion. The size network is not aware of whether the features are from upsampled or downsampled feature maps.

**Comment by mks0601** [comment]
yes. and the output is also not in any absolute space as the values are directly regressed.
